YurisCodingClub/pro-check
# Added Header component

## Description



- Created `Header.js` in the `src/components` folder and imported it in `App.js` 
- Created header component using semantic HTML `<header></header>` tag for better accessibility
- I had to add some styles in `Header.js` for better visibility
- I had to add some `margin-top` in the `Code.js` component to get some space for the `Header.js` components. Otherwise, it was overlapping with one another
- I added an `svg logo` as a placeholder so that when the `logo` is comfired, it can be changed later
- I have used the `position: fixed` and `top:0` property in the style of the `Header` component so that it stays at top when we scroll the page


## Related Issue


#2 

## Acceptance Criteria



 - [x] Create a component called Header.js and save it in a folder called components in the src directory
 - [x]  A header is shown with a logo and the name in it
 - [x] The header is imported and displayed in the App.js component
 - [x] Only the header is displayed
 - [ ] Plain HTML, no styling added
 - [x] The header meets accessibility standards like semantic HTML
 - [x] The header is fixed (stays at its position even when scrolling)

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|✓| :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before



![image](https://user-images.githubusercontent.com/47440165/199044742-0379bbd6-3930-49b1-96e6-c161df27e6c3.png)


### After




![image](https://user-images.githubusercontent.com/47440165/199045043-f721a16f-dc75-48a6-af2a-3c0386e9bab0.png)


## Testing Steps / QA Criteria


- Go to `src/components/Header.js` to check the newly added component
- You can also view the deploy preview which will be provided by netlify
